app/src/main/java/de/danoeh/antennapod/adapter/SubscriptionsRecyclerAdapter.java

```diff
@@ -0,0 +1,268 @@
+        holder.onBind(drawerItem, position);
+        holder.itemView.setOnCreateContextMenuListener(this);
+        if (inActionMode()) {
+            if (isFeed) {
+                holder.selectCheckbox.setVisibility(View.VISIBLE);
+                holder.selectView.setVisibility(View.VISIBLE);
+            }
+            holder.selectCheckbox.setChecked((isSelected(position)));
+            holder.selectCheckbox.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
+                @Override
+                public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
+                    setSelected(position, isChecked);
+                }
+            });
+        } else {
+            holder.selectView.setVisibility(View.GONE);
+        }
+
+        holder.itemView.setOnLongClickListener(v -> {
+            if (!inActionMode()) {
+                if (isFeed) {
+                    selectedFeed = ((NavDrawerData.FeedDrawerItem) drawerItem).feed;
+                    longPressedPosition = position;
+                } else {
+                    selectedFeed = null;
+                }
+            }
+            return false;
+        });
+
+        holder.itemView.setOnClickListener(v -> {
+            if (drawerItem == null) {
+                return;
+            } else if (isFeed) {
+                if (inActionMode()) {
+                    holder.selectCheckbox.setChecked(!isSelected(position));
+                } else {
+                    Fragment fragment = FeedItemlistFragment
+                            .newInstance(((NavDrawerData.FeedDrawerItem) drawerItem).feed.getId());
+                    mainActivityRef.get().loadChildFragment(fragment);
+                }
+                return;
+            } else if (!inActionMode()) {
+                Fragment fragment = SubscriptionFragment.newInstance(drawerItem.getTitle());
+                mainActivityRef.get().loadChildFragment(fragment);
+            }
+
+        });
+
+    }
+
+    @Override
+    public int getItemCount() {
+        return listItems.size();
+    }
+
+    @Override
+    public long getItemId(int position) {
+        NavDrawerData.DrawerItem drawerItem = listItems.get(position);
+        if (drawerItem != null) {
+            return drawerItem.id;
+        }
+        return RecyclerView.NO_ID;
+    }
+
+    @Override
+    public void onCreateContextMenu(ContextMenu menu, View v, ContextMenu.ContextMenuInfo menuInfo) {
+        if (selectedFeed != null && !inActionMode()) {
+            MenuInflater inflater = mainActivityRef.get().getMenuInflater();
+            inflater.inflate(R.menu.nav_feed_context, menu);
+            menu.setHeaderTitle(selectedFeed.getTitle());
+        }
+    }
+
+    public boolean onContextItemSelected(MenuItem item) {
+        if (item.getItemId() == R.id.multi_select) {
+            startSelectMode(longPressedPosition);
+            return true;
+        }
+        return false;
+    }
+
+
```

Comment:
Please consistently use one line between methods. There are a few places where you use 2 (not just here)

Reply:
k